src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Ball winner = new Ball();
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        winner.setColor(colorSupplier.getRandomColor());
+        winner.setNumber(random.nextInt(100));

```

Comment:
Magic numbers should be avoided. Instead, the maximum number for 'nextInt' should be defined as a constant field with a clear name indicating its purpose.